### DISCUSSION TITLE
累計作業時間をリセットする機能の利点

### DISCUSSION THREAD

**Discussion by kani3camp** [sorarideblog/youtube-study-space#62] [Q&A] [answered]
累積作業時間データは、この作業部屋システムで最も重要な個人データです。
できるだけ自動でリセットする機能はつけたくないというのが本音です。

「累積作業時間をリセットする機能がほしい」という意見もあります。
皆さんはどう思いますか？
また、累積作業時間をリセットする利点は何があると思いますか？

**Comment by window794** [comment, marked as answer]
累積時間のリセットを希望する人は、期間ごととか、勉強内容ごと？（例えばこの期間からこの期間の○○試験の勉強とか）で集計したという思いがあるのかもしれませんね。
